pkg/controller/iam/partialpolicy/iampartialpolicy_controller.go

```diff
@@ -171,6 +174,15 @@ func (r *ReconcileIAMPartialPolicy) Reconcile(ctx context.Context, request recon
 	return reconcile.Result{RequeueAfter: jitteredPeriod}, nil
 }
 
+func (r *ReconcileIAMPartialPolicy) handleDefaultStateIntoSpecValue(pp *iamv1beta1.IAMPartialPolicy) error {
```

Comment:
I note that we don't have the call to `EnsureSpecIntoStateAnnotation` here in these reconcilers.  Should that have been here previously?

Reply:
So in #1151 I think you suggested we just forgot; so that's fine.  I was just worried there was some reason...

Reply:
@diviner524 do you happen to have any context about it? I personally don't know why the backwards compatible logic is not there for IAM resources.